internal/cache/cachedb.go

```diff
@@ -25,6 +25,7 @@ const (
 	defaultCachePath = "/var/lib/aad/cache"
 	passwdDB         = "passwd.db" // root:root 644
 	shadowDB         = "shadow.db" // root:shadow 640
+	dbConnArgs      = "?_journal_mode=wal"
```

Comment:
After the change, the `=` became misaligned. Can you fix it to calm down the linter?

Reply:
Yes, thanks for catching that. :)
fixed in latest commit, I also updated the branch.